src/components/Header/Header.js

```diff
@@ -1,40 +1,40 @@
-import React, { useState } from "react";
-import { useDispatch } from "react-redux";
-import { Link } from "react-router-dom";
-import { FaAngleLeft } from "react-icons/fa";
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { FaAngleLeft } from 'react-icons/fa';
 import {
   fetchAsyncMovies,
   fetchAsyncShows,
-} from "../../features/movies/movieSlice";
-import user from "../../images/user.png";
-import "./Header.scss";
+} from '../../redux/movies/movieSlice';
+import './Header.scss';
 
 const Header = () => {
-  const [ term, setTerm ] = useState('');
+  const [term, setTerm] = useState('');
   const dispatch = useDispatch();
   const submitHandler = (e) => {
     e.preventDefault();
     dispatch(fetchAsyncMovies(term));
     dispatch(fetchAsyncShows(term));
-    setTerm("");
-  }
+    setTerm('');
+  };
   return (
     <div className="header">
-        <div className="logo">
-         <Link to="/"><FaAngleLeft className="fa-icon"/></Link>
-        </div>
-        <div className="search-bar">
-          <form onSubmit={submitHandler}>
-            <input 
-            type='text' 
-            value={term} 
-            placeholder='search for movies' 
-            onChange={(e) => setTerm(e.target.value)}/>
-            <button type="submit">
-              <i className="fa fa-search"></i>
-              </button>
-          </form>
-        </div>
+      <div className="logo">
+        <Link to="/"><FaAngleLeft className="fa-icon" /></Link>
+      </div>
+      <div className="search-bar">
+        <form onSubmit={submitHandler}>
+          <input
+            type="text"
+            value={term}
+            placeholder="search for movies"
+            onChange={(e) => setTerm(e.target.value)}
+          />
+          <button type="submit">
+            <i className="fa fa-search" />
+          </button>
+        </form>
+      </div>
     </div>
   );
 };
```

Comment:
- [ ] I think your search doesn't work correctly. Data is not retrieved immediately when entering a given parameter like the movie name. when I input the letter in the search bar and click on the search button, I got the error in the screenshot below. Let's kindly use the ES6 filter() to filter data from the API.
![Screenshot from 2023-01-13 04-46-10](https://user-images.githubusercontent.com/63087491/212234346-87e0bd9a-f807-4b83-86be-2761f9971a6c.png)